functional

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh -e
```

Comment:
Please just move this script into the tools/functional-tester directory, it doesn't belong at the top level of the repo